### ISSUE TITLE
addition of orderOnline to Restaurant types

### ISSUE THREAD

**Issue by dkutcher** [schemaorg/schemaorg#1796]
In a number of instances it is becoming necessary to define an optional URL to specify the URL of a restaurant's online ordering system, which may be either internal, sub-domain, or defined 3rd party provider.

-David

**Comment by RichardWallis** [comment]
Would not [contactPoint](http://schema.org/contactPoint) satisfy this need?
```
<script type="application/ld+json">
{
  "@context": "http://schema.org",
  "@type": "Restaurant",
  "name": "Local Chinese",
  "contactPoint": {
         "@type": "ContactPoint",
         "name": "Online",
         "url": "http://order.localchinese.example.com",
         "telephone": "08000 123 12345",
         "areaServed": "London",
         "hoursAvailable": {
              "@type": "OpeningHoursSpecification",
              "closes":  "23:00:00",
              "opens":  "17:00:00"
          }
    }
```

**Comment by dkutcher** [comment]
"A contact point for a person or organization. Supersedes contactPoints."

Unsure, and at first glance seems to be left up to interpretation. I would put it on the level as "acceptsReservations" which then links to a reservation system (such as OpenTable). This would operate similarly, but I do like the hoursAvailable addition to the concept as well as areaServed.

**Comment by chaals** [comment]
I don't think `contactPoint` covers this adequately - it might equally be a point to reserve a table, to ask whether my umbrella was left under one, or just to say how great the flower arrangements were.

The use case feels much more like a [potentialAction](https://schema.org/potentialAction) ...

**Comment by RichardWallis** [comment]
You are right @chaals contactPoint(s) would be listed for general enquiries, table reservations, etc.

Another way to look at the use case might be to see it as a [Service](http://schema.org/Service) offered by the Restaurant.   In fact if it was the online order point for a delivery service, that is exactly how it would be done.

**Comment by chaals** [comment]
Perhaps specifically the answer is an `[availableChannel](http://schema.org/availableChannel)` of a `[FoodService](http://schema.org/FoodService)`?

This seems like an area we should clean up... What is the relationship between a Service and a PotentialAction?

**Comment by vickitardif** [comment]
I agree with @chaals. It is worth considering how schema.org/FoodService and schema.org/OrderAction work together before trying to modify schema.org/ContactPoint.
